views/js/sample.js

```diff
@@ -0,0 +1,43 @@
+var googleUser = {};
+  var startApp = function() {
+    gapi.load('auth2', function(){
+      // Retrieve the singleton for the GoogleAuth library and set up the client.
+      auth2 = gapi.auth2.init({
+        client_id: '594743792021-tvffn11n961cqea8mbufm9gnts372m0n.apps.googleusercontent.com',
+        cookiepolicy: 'single_host_origin',
+      });
+      attachSignin(document.getElementById('customBtn'));
+    });
+  };
+  var startApp2 = function() {
+    gapi.load('auth2', function(){
+      // Retrieve the singleton for the GoogleAuth library and set up the client.
+      auth2 = gapi.auth2.init({
+        client_id: '594743792021-tvffn11n961cqea8mbufm9gnts372m0n.apps.googleusercontent.com',
+        cookiepolicy: 'single_host_origin',
+      });
+      attachSignin2(document.getElementById('customBtn2'));
+    });
+  };
+
+  function attachSignin(element) {
```

Comment:
Can't we use a common function fo startApp and attachSignin and just pass
document.getElementById('customBtn'), document.getElementById('customBtn2'),  document.getElementById('name') document.getElementById('name2') respectively to the functions as the object. as the function names are exactly same but the only difference is of element,